packages/plugins/other/visitor-plugin-common/src/client-side-base-visitor.ts

```diff
@@ -204,21 +223,25 @@ export class ClientSideBaseVisitor<TRawConfig extends RawClientSideBasePluginCon
   }
 
   protected _gql(node: FragmentDefinitionNode | OperationDefinitionNode): string {
+    const fragments = this._transformFragments(node);
     const doc = this._prepareDocument(`
     ${
       print(node)
         .split('\\')
         .join('\\\\') /* Re-escape escaped values in GraphQL syntax */
     }
-    ${this._includeFragments(this._transformFragments(node))}`);
+    ${this._includeFragments(fragments)}`);
 
     if (this.config.documentMode === DocumentMode.documentNode) {
       const gqlObj = gqlTag(doc);
-
       if (gqlObj && gqlObj['loc']) {
         delete gqlObj.loc;
       }
-
+      if (this.config.importDocumentNodeFragments && fragments.length > 0) {
+        const fragmentsSpreads = fragments.filter((name, i, all) => all.indexOf(name) === i).map(name => `...${name}.definitions`);
+        const definitions = [...gqlObj.definitions.map(JSON.stringify), ...fragmentsSpreads].join();
+        return `{"kind":"${Kind.DOCUMENT}","definitions":[${definitions}]}`;
```

Comment:
It's a JS object, so no need for `"`

Reply:
I mainly used this format to keep it consistent with the rest of the stringified document since it's using `JSON.stringify`, and also to keep it consistent with the default `documentNode` mode which outputs the document with `"`. thoughts?